server/s3_common_utilities.cc

```diff
@@ -126,6 +126,39 @@ bool stoi(const std::string &str, int &value) {
   return isvalid;
 }
 
+void size_based_bucketing_of_objects(std::string &oid_str, size_t obj_size){
+  s3_log(S3_LOG_INFO, ""  , "Entering\n");
+  size_t object_size_for_bucketing = obj_size;
+
```

Comment:
good to have a comment at begining, where you mention range and prefix
Object size.                                           Prefix
========                                          =====
 less than  1K                                          I
 less than 10KB and ...                           H

Also mention that this is done as listing of key's in motr happens in lexographic order, hence bigger object will get listed first.

Reply:
Done.